### ISSUE TITLE
No item found.

### ISSUE THREAD

**Issue by usernoobguy** [Wouterdek/magic4pc#18]
<img width="441" alt="image" src="https://user-images.githubusercontent.com/99551629/170110630-ef7cebe3-b242-4bba-87f9-55d081a01085.png">
My laptop can’t find out TV at magic4pc main menu, so I tried to turn off my laptop firewall and make sure they all on same local net, but it won’t works, pls help I can’t figure out this issue so long😭😭😭😭

**Comment by usernoobguy** [comment]
Is it still available 🥲?

**Comment by git-d333p** [comment]
for me the problem was the firewall. worked after that. Im just trying to figure out, how to get pause/play with that main middle button on the remote.

**Comment by usernoobguy** [comment]
> for me the problem was the firewall. worked after that. Im just trying to figure out, how to get pause/play with that main middle button on the remote.

It’s still not working after I turned off my pc firewall, I’m almost give up…..

**Comment by git-d333p** [comment]
I had 2 entries of magic4pc in my firewall. activated all, after that it worked. i had the tv on with the tv app running, the pc with the client running (and searching), activated everything in the firewall and the client immediately found my tv.

**Comment by usernoobguy** [comment]
> I had 2 entries of magic4pc in my firewall. activated all, after that it worked. i had the tv on with the tv app running, the pc with the client running (and searching), activated everything in the firewall and the client immediately found my tv.

Yeah i tried… btw, when i launch magic4pc on my LG TV, it always in black screen, is that normal?

**Comment by git-d333p** [comment]
maybe if you didnt chose the right hdmi channel. if you cant chose anything i would reinstall the ipk.

**Comment by usernoobguy** [comment]
> maybe if you didnt chose the right hdmi channel. if you cant chose anything i would reinstall the ipk.

**Comment by usernoobguy** [comment]
> maybe if you didnt chose the right hdmi channel. if you cant chose anything i would reinstall the ipk.
here is a video to explain my issue

https://user-images.githubusercontent.com/99551629/173917908-92824ee6-b194-485d-888c-ee6e8f06dca9.MOV

**Comment by usernoobguy** [comment]
> maybe if you didnt chose the right hdmi channel. if you cant chose anything i would reinstall the ipk.

Sadly, still no idea🥹

**Comment by dinbtechit** [comment]
Same issue here - magic4pc server (on the TV) just simply displays a blank screen. On the windows - magic4pc say "No items found".

@usernoobguy  Did you find a solution for this?

magic4linux can't find the server either on my linux machine. I feel like the server is probably not running. I have not rooted the tv but installed it using the developer mode. Not sure if that makes any difference.
